MapboxDirections/MBVisualInstruction.swift

```diff
@@ -41,41 +41,53 @@ open class VisualInstruction: NSObject, NSSecureCoding {
      */
     @objc public let secondaryTextComponents: [VisualInstructionComponent]?
     
+    
+    /**
+     :nodoc:
+     Indicates what side of a bidirectional road the driver must be driving on. Also referred to as the rule of the road.
+     */
+    @objc public var drivingSide: DrivingSide
+    
     /**
      :nodoc:
      Initialize a `VisualInstruction` from a dictionary.
      */
-    @objc public convenience init(json: [String: Any]) {
+    @objc public convenience init(json: [String: Any], drivingSide: DrivingSide) {
```

Comment:
Question, could we have an additional convenience init which specifies driving side, this way we can offer 2 initialization types.